x/bank/app_test.go

```diff
@@ -99,39 +102,29 @@ type suite struct {
 	BankKeeper         bankkeeper.Keeper
 	AccountKeeper      types.AccountKeeper
 	DistributionKeeper distrkeeper.Keeper
-	App                *runtime.App
+	App                *simapp.SimApp // use simapp instead for tests since depinject is not supported yet for heimdall app initialization
```

Comment:
Fair enough, you might even consider just replacing with a minimal interface here for the purposes of testing. In future this test will be refactored out to an e2e directory

Reply:
What minimal interface are you talking about ?

Reply:
Instead of declaring that App must implement SimApp (which is the kitchen sink), you might consider creating a lighter weight test app interface. Just a suggestion but if the plan is to revert to runtime.App then nvm.

Reply:
We haven't come to a decision whether we will be switching to depinject powered app. But I think we can keep simapp for now. If in future if the SDK testing package provides both ways of initializing an app (it should IMHO unless I missed it), then we can straight out swap simapp.

Reply:
Just make sure that there's not interfaces in runtime.App unrelated to depinject that are missing from Simapp

Reply:
I'm not sure I get that. What would be the consequence ?